deploy/v2.0.0-alpha.1/csi-azuredisk-scheduler-extender.yaml

```diff
@@ -44,7 +44,7 @@ spec:
       labels:
         app: azdiskschedulerextender
```

Comment:
Should we change the deployment, pod and labels to `csi-azuredisk-scheduler-extender` as well?

Reply:
Yeah, that would be a good idea.